index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Kyle Hillman's Portfolio</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles/style.css" />
+    <link rel="stylesheet" href="styles/fonts.css" />
+    <link rel="stylesheet" href="styles/reset.css" />
+  </head>
+  <body>
+    <ul class="navigation">
+      <li class="nav-item">Projects</li>
+      <li class="nav-item">Github</li>
+      <li class="nav-item">About</li>
+    </ul>
+    <input type="checkbox" id="nav-trigger" class="nav-trigger" />
+    <label for="nav-trigger" class="icon-menu"></label>
+    <main class="site-wrap">
+      <h1>Kyle Hillman Portfolio</h1>
+      <img />
+      <img />
+      <img />
+      <div>
+        <h2>Skills</h2>
+        <ul>
+          <li>Skill 1</li>
+          <li>Skill 2</li>
+          <li>Skill 3</li>
+          <li>Skill 4</li>
+        </ul>
+        <ul>
+          <li>Skill 5</li>
+          <li>Skill 6</li>
+          <li>Skill 7</li>
+          <li>Skill 8</li>
+        </ul>
+      </div>
+      <div>
```

Comment:
As above, rather than putting your About section inside a `<div>`, you want to use a more semantic tag, like `<section id="about">`